### PR TITLE
Update based on comments

### PR DESCRIPTION
**Pull request by qiujian16**
- One-line PR description:

- Issue link:

- Other comments:

### CODE REVIEW

**Comment by RainbowMango** [comment]
Awesome, the last comment posted at https://github.com/qiujian16/k8s-enhancements/pull/3#issuecomment-1782234631.

**Comment by qiujian16** [comment]
@RainbowMango I've updated the member cluster terminology. I try to scan and I think I change all the terminology in the doc, would you also double check?

File: keps/sig-multicluster/cluster-inventory/README.md [hunk 1]
```diff
@@ -294,6 +294,11 @@ the API proposed by this KEP aims to
   cluster inventory API for the purpose of workload distribution, operation
   management etc. The consumer SHOULD not create this API resource.
 
+- **Member Cluster**: A kubernetes cluster that is managed by the cluster
+manager. A cluster manager SHOULD have sufficient permission to access
+the member cluster to fetch the information so it can update the status
+of the cluster inventory API resource.
```
Comment (RainbowMango) [inline keps/sig-multicluster/cluster-inventory/README.md:300]: Awesome.

Only one `spoke cluster` left in the description of `Cluster Manager` on above.
https://github.com/qiujian16/k8s-enhancements/pull/4/files#diff-79aeff7187b5ffc40b00fb82b65aba43131074801edc7e26283a5f833a226420R286-R287

Comment (qiujian16) [reply keps/sig-multicluster/cluster-inventory/README.md:300]: updated

**Comment by RainbowMango** [comment]
LGTM. Thanks!!!

**Pull request merged**